src/components/helpcenter/HelpCenterMenuContent.vue

```
@@ -1,38 +1,54 @@
 <template>
   <div
-    class="help-center-menu"
+    class="help-center-menu flex flex-col items-start gap-1"
     role="menu"
-    :aria-label="$t('helpCenter.helpFeedback')"
+    aria-label="Help Center Menu"
```

ESLint

[error] raw text 'Help Center Menu' is used

(@intlify/vue-i18n/no-raw-text)